azure-pipelines-release.yml

```diff
@@ -0,0 +1,730 @@
+parameters:
+- name: FindApprenticeshipTraining
+  type: boolean
+  default: false
+- name: Reservations
+  type: boolean
+  default: false
+- name: EmployerIncentives
+  type: boolean
+  default: false
+- name: FindEpao
+  type: boolean
+  default: false
+
+trigger: none
+
+pr: none
+
+variables:
+- group: Release Management Resources
+- group: Wildcard Cert Variables v2
+
+resources:
+  repositories:
+  - repository: self
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/tags/0.2.1
+    endpoint: GitHub (SFA)
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.2.0
+    endpoint: GitHub (SFA)
+  pipelines:
+  - pipeline: das-employer-config
+    project: Digital Apprenticeship Service
+    source: das-employer-config
+    branch: master
```

Comment:
Should this take a parameter (that defaults to master) to allow a branch of config to be used (as we do with most classic releases)?

Reply:
Not sure if its possible to pass a pipeline branch input in as a parameter as I'm getting YAML failures.

Its one of the kinks of YAML release pipelines that hasn't been implemented yet https://developercommunity.visualstudio.com/content/problem/1064010/yaml-pipelines-variable-pipeline-resources.html

Reply:
I should've checked the docs, this is possible out of the box without any template changes - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#manual-version-picker-for-resources-in-the-create-run-dialogue